Formula/chkrootkit.rb

```diff
@@ -1,9 +1,8 @@
 class Chkrootkit < Formula
   desc "Rootkit detector"
   homepage "http://www.chkrootkit.org/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
```

Comment:
ugh, even with the MS nat pro, I seem to have flubbed that line, I just edited it in the pull rather than in git, sorry. right hand trouble today.

corrected now.